com.unity.formats.alembic/Runtime/Scripts/Exporter/AlembicExporter_impl.cs

```diff
@@ -417,9 +417,13 @@ class TransformCapturer : ComponentCapturer
 
             public override void Setup(Component c)
             {
-                var target = c as Transform;
-                abcObject = parent.abcObject.NewXform(target.name + " (" + target.GetInstanceID().ToString("X8") + ")", timeSamplingIndex);
-                m_target = target;
+                if (parent != null && c is Transform)
```

Comment:
When parent can be null ? and when can setup of a TransformCapturer be called with something that is not a Transform ?
Could this be also an issue with all Capturers ?

Reply:
The parent can be null because it was deleted. 
The parent cannot be different than a transform really. I am just checking my assumptions. Previous code was casting and never checking the result.